ReactNativeTraining/src/app/screens/home/groups/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+import data from '../../../../utils/constants';
+import ContactList from '../../components/ContactList/index';
+
+function Groups() {
+  return <ContactList data={data.groups} />;
+}
+
+export default Groups;
```

Comment:
Can u move groups, chats and contacts screens to  `screens/Home/screens/...`? For example, `screens/home/groups/index.js` -> `screens/home/screens/groups/index.js` this components are also screens